templates/registration/profile.html

```diff
@@ -13,15 +13,13 @@ <h2 class="profile-header">{% firstof user.get_full_name user.get_username %}</h
 
     {% if artists %}
         <ul class="tabs" data-tabs id="profile-tabs">
-            <li class="tabs-title is-active"><a href="#updates"><i class="fa fa-newspaper-o fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Activity</span></a></li>
-            <li class="tabs-title"><a href="#profile-stats"><i class="fa fa-bar-chart fa-2x tab-icon" aria-hidden="true"></i><span class="profile-name">Earnings</span></a></li>
```

Comment:
If we're removing this tab, then we should remove the data from the page too, shouldn't we?

Reply:
Ahh.. yes